sklearn/mixture/gmm.py

```diff
@@ -646,8 +765,7 @@ def aic(self, X):
         -------
         aic: float (the lower the better)
         """
-        return - 2 * self.score(X).sum() + 2 * self._n_parameters()
-
+        return super(GMM, self).aic(X).sum() / X.shape[0]
```

Comment:
```python
def aic(self, X):
"""[...]"""
return -2 * self.score(X) * X.shape[0] + 2 * self._n_parameters()
```

what is this `.sum()` for? isn't `super(GMM, self).aic(X)` a float?
Also you should not divide `2 * self._n_parameters()` by `X.shape[0]`